__docs__/wonder-blocks-card/card.stories.tsx

```diff
@@ -0,0 +1,306 @@
+                            to dismiss.
+                        </BodyText>
+                    </Card>
+                </View>
+            );
+        };
+
+        return <CardWithRef />;
+    },
+};
+
+export const WithTag: StoryComponentType = {
+    parameters: {
+        chromatic: {disableSnapshot: true}, // disable snapshot since doesn't have a meaningful visual effect
+    },
+    render: () => (
+        <View tag="ul">
+            <Card tag="li" styles={{root: styles.card}}>
+                <Heading>Card 1</Heading>
+            </Card>
+            <Card tag="li" styles={{root: styles.card}}>
+                <Heading>Card 2</Heading>
+            </Card>
+        </View>
+    ),
+};
+
+export const InAStack: StoryComponentType = {
+    parameters: {
+        chromatic: {disableSnapshot: true}, // disable snapshot since doesn't have a meaningful visual effect
+    },
+    render: () => (
+        <>
+            <Heading>Cards in a stack</Heading>
+            <View tag="ul">
+                <View tag="li" style={styles.stackedCard}>
+                    <Card styles={{root: styles.card}}>
+                        <Heading>Active Card</Heading>
+                        <Button>CTA</Button>
+                    </Card>
+                </View>
+                <View tag="li" style={[styles.stackedCard, styles.dimmed]}>
+                    <Card inert={true} styles={{root: styles.card}}>
+                        <Heading>Inactive Card</Heading>
+                        <Button>CTA</Button>
+                    </Card>
+                </View>
+            </View>
+        </>
+    ),
+};
+
+export const InAGrid: StoryComponentType = {
+    parameters: {
+        chromatic: {disableSnapshot: true}, // disable snapshot since doesn't have a meaningful visual effect
+    },
+    render: () => (
+        <>
+            <Heading>Cards in a grid</Heading>
+            <View
+                style={[
+                    styles.gemRow,
+                    {flexWrap: "wrap", gap: spacing.large_24},
+                ]}
+            >
+                <Card styles={{root: styles.card}}>
+                    <Heading>Card 1</Heading>
+                    <BodyText>This is a basic card.</BodyText>
+                </Card>
+                <Card styles={{root: styles.card}}>
+                    <Heading>Card 2</Heading>
+                    <BodyText>This is a basic card.</BodyText>
+                </Card>
+                <Card styles={{root: styles.card}}>
+                    <Heading>Card 3</Heading>
+                    <BodyText>This is a basic card.</BodyText>
+                </Card>
+                <Card styles={{root: styles.card}}>
+                    <Heading>Card 4</Heading>
+                    <BodyText>This is a basic card.</BodyText>
+                </Card>
+            </View>
+        </>
+    ),
+};
+
+export const WithStyleProps: StoryComponentType = {
```

Comment:
Do we want to add documentation around styling cards? I think we prefer consumers to use the dedicated props (borderRadius, backgroundColor, etc) over `styles` where possible so only a subset of options are available!

Reply:
Yes, I added some more docs!